CuratedContent/WorkingEffectivelyWithLegacyCode.md

```diff
@@ -76,7 +77,7 @@ Another major contribution of this book is in detailing the strategies that can
   - **Object Seams**: Define interfaces and replace production objects with mock or fake objects in a test harness. (NOTE: Prefer object seams to link or preprocessing seams!)
 
 The Legacy Code Change Algorithm is applied in many small iterations over and over again as pieces of the legacy software are changed.
-Over time, these small refactorings can result in significant improvements to the quality and sustainability of the software and can even perform major (beneficial) architectural changes over time.
+Over time, all the refactoring can result in significant improvements to the quality and sustainability of the software and can even perform major (beneficial) architectural changes over time.
```

Comment:
Again, it seems more precise to say "small small refactorings" vs. "all of the refactoring".  That seems to be more standard in the SE refactoring literature.